### PR TITLE
009 decompte

### PR DESCRIPTION
**Pull request by Hamza-Luciano**

### CODE REVIEW

**Review by FrancoisSalmon — DISMISSED** [review]
ok for me

File: frontend/src/components/counter/index.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          <img className="billeterie" src={billeterie} alt="billeterie" />
+        </a>
+      </div>
+    </SMyTimer>
+  );
+}
+
+export default function App() {
+  const time = new Date();
+  time.setSeconds(time.getSeconds() + 691200); // 8 jours : obligation de convertir la durée en secondes, mais pb quand actualisation de la page
```
Comment (LuigiMenez) [inline frontend/src/components/counter/index.jsx:58]: Comments in English please

File: frontend/src/components/counter/index.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        </ul>
+        <span>-</span>
+        <ul>
+          <li className="numbersCount"> {minutes} </li>
+          <li>MINUTES</li>
+        </ul>
+      </div>
+      <div className="billetDiv">
+        <a href="https://www.sah-ph.fr/billetterie/">
+          <img className="billeterie" src={billeterie} alt="billeterie" />
```
Comment (LuigiMenez) [inline frontend/src/components/counter/index.jsx:49]: className in English

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Comments and className in English

**Review by FrancoisSalmon — APPROVED** [review]
no more english

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/components/counter/index.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      <div className="billetDiv">
+        <a href="https://www.sah-ph.fr/billetterie/">
+          <img className="boxOffice" src={boxOffice} alt="billeterie" />
+        </a>
+      </div>
+    </SMyTimer>
+  );
+}
+
+export default function App() {
```
Comment (LoicBrassart) [inline frontend/src/components/counter/index.jsx:56]: Wtf ? Why do you export a second app ? o.O

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- Evils to fix
- Component folders (here `components/counter`) should be PascalCase (`components/Counter`)

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix conflict

**Review by FrancoisSalmon — DISMISSED** [review]

File: frontend/src/components/counter/index.jsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import PropTypes from "prop-types";
+import { useTimer } from "react-timer-hook";
+import boxOffice from "@assets/boxOffice.png";
+import logoSah from "@assets/Logo_SAH.png";
+import SMyTimer from "./style";
+
+function MyTimer({ expiryTimestamp }) {
+  const { minutes, hours, days } = useTimer({
+    expiryTimestamp,
+    onExpire: () => console.warn("onExpire called"),
```
[the remark this answers is not in the document]
Comment (LuigiMenez) [reply frontend/src/components/counter/index.jsx:11]: same

File: frontend/src/components/counter/index.jsx (hunk 3 above)
Comment (LuigiMenez) [reply frontend/src/components/counter/index.jsx:56]: same

File: frontend/src/components/counter/index.jsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+          <img className="boxOffice" src={boxOffice} alt="billeterie" />
+        </a>
+      </div>
+    </SMyTimer>
+  );
+}
+
+export default function App() {
+  const time = new Date();
+  time.setSeconds(time.getSeconds() + 691200); // = 8 days, mandatory conversion to seconds to program the timer
```
[the remark this answers is not in the document]
Comment (LuigiMenez) [reply frontend/src/components/counter/index.jsx:58]: same

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Still the same worries as Loic asked you

**Pull request closed without merging**